rxbinding-design/src/main/java/com/jakewharton/rxbinding/support/design/widget/RxTextInputLayout.java

```diff
@@ -0,0 +1,76 @@
+package com.jakewharton.rxbinding.support.design.widget;
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.support.design.widget.TextInputLayout;
+import rx.functions.Action1;
+
+/**
+ * Static factory methods for creating {@linkplain Action1 actions} for {@link TextInputLayout}.
+ */
+public final class RxTextInputLayout {
+
+  /**
+   * An action which sets the counterEnabled property of {@code view} with a boolean value.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   */
+  @CheckResult @NonNull
+  public static Action1<? super Boolean> counterEnable(@NonNull final TextInputLayout view) {
```

Comment:
So we're pretty inconsistent with how these action methods are named. Right now there's one called "check" and another called "checked" on two different types which do the same thing. The README isn't super clear on a naming scheme. I think we should probably favor matching the method name just without the "set" prefix which would make this "counterEnabled".